travel/src/main/java/com/tx/travel/security/jwt/JwtUtils.java

```diff
@@ -75,9 +86,18 @@ public boolean validateJwtToken(String authToken) {
     return false;
   }
 
-  public String generateTokenFromUsername(String username) {
+  public String generateTokenFromUser(UserDetailsImpl userPrincipal) {
+    Map<String, Object> claims = new HashMap<>();
```

Comment:
can they be String?

Reply:
.setClaims() from Jwts package wouldn't be able to resolve Map<String,String> , accepts only Map<String, Object>